src/scss/base/_base.scss

```diff
@@ -9,24 +9,30 @@
 
 h1 {
 	color: $grey-dark;
-	font-size: rem(48);
+	font-size: rem(35);
 	font-weight: $font-weight-semibold;
 }
 
 h2 {
 	color: $grey-dark;
-	font-size: rem(48);
+	font-size: rem(24);
 	font-weight: $font-weight-medium;
 }
 
 h3,
 .h3 {
-	font-size: rem(24);
+	font-size: rem(18);
 	font-weight: $font-weight-bold;
 }
 
-p {
+h4 {
 	font-size: rem(16);
+	font-weight: $font-weight-bold;
+}
+
+p {
+	font-size: rem(14);
+	font-weight: $font-weight-normal;
```

Comment:
Is it necessary to set `font-weight` to normal? Isn't it the browser default?

Reply:
It is. This line has been removed